pywren/executor.py

```diff
@@ -146,13 +146,20 @@ def map(self, func, iterdata, extra_env=None, extra_meta=None,
             invoke_pool_threads=64, data_all_as_one=True,
             use_cached_runtime=True, overwrite_invoke_args=None, exclude_modules=None):
         """
-        # FIXME work with an actual iterable instead of just a list
-
-        data_all_as_one : upload the data as a single object; fewer
-        tcp transactions (good) but potentially higher latency for workers (bad)
-
-        use_cached_runtime : if runtime has been cached, use that. When set
-        to False, redownloads runtime.
+        :param func: the function to map over the data
+        :param iterdata: An iterable of input data to the mapped function
+        :param extra_env: foo
```

Comment:
can complete these parameters?  It will be nice to include the default values again.